sphinx/source/docs/user_guide/embed.rst

```diff
@@ -369,6 +370,35 @@ will look something like this:
 
 For full details read the autoload_server reference here: |autoload_server|.
 
+Alternatively, two other methods allow to embed content from a bokeh server
+in a similar fashion to ``autoload_server`` but with some subtle differences:
+with ``server_document`` a new session will systematically be generated and
+an entire document will be returned; with ``server_session`` an existing session
+id and model must be provided. Another difference from ``autoload_server`` is
+that with those two methods one may choose to not load the JS/CSS resource
+files by passing a ``resources="none"`` parameter.
+
+Here is an example using ``server_document``:
+
+.. code-block:: python
+
+    from bokeh.embed import server_document
+    script = server_document(url="https://demo.bokehplots.com/apps/slider")
```

Comment:
I think this can be passed as a positional argument now
```
server_document("https://demo.bokehplots.com/apps/slider")
```
which I think is preferable to demonstrate.